x-pack/legacy/plugins/uptime/common/constants/plugin.ts

```diff
@@ -5,6 +5,7 @@
  */
 
 export const PLUGIN = {
+  APP_ROOT_ID: 'react-uptime-root',
   ID: 'uptime',
   ROUTER_BASE_NAME: '/app/uptime#/',
```

Comment:
Do we still need ID?

Reply:
The `ID` field is used when the server is bootstrapped, so we are still using it. Perhaps we should rename `APP_ROOT_ID`, or move it to a different file. This constant object does have fields used on the front- and backend though, so I figured it was ok here.